internal/key/key.go

```diff
@@ -165,6 +165,25 @@ func matchFilesAndDirs(filename string, recursive bool) ([]string, []string, err
 	matchedDirs := []string{}
 	matchedFiles := []string{}
 
+	// Check if the filename is a specific relative path (contains directory separators)
```

Comment:
I think this is fine - just double checking we don't want to check if for a `startsWith("./"` type deal-io

Reply:
@mitchbne There are a few ways to do this but there are lots of pitfalls supporting it cross platform so I am going to focus on ensuring we have lots of test cases then figure out later. 

Hopefully that makes sense.